sycl/include/sycl/feature_test.hpp.in

```diff
@@ -74,6 +73,7 @@ __SYCL_INLINE_VER_NAMESPACE(_V1) {
 #cmakedefine01 SYCL_BUILD_PI_CUDA
 #if SYCL_BUILD_PI_CUDA
 #define SYCL_EXT_ONEAPI_BACKEND_CUDA 1
+#define SYCL_EXT_ONEAPI_BFLOAT16_MATH 1
```

Comment:
Is this SYCL_EXT_ONEAPI_BFLOAT16_MATH or SYCL_EXT_ONEAPI_BFLOAT16. The math document says the latter. can you please take a look?